Kickstarter-iOS/ViewModels/SettingsCurrencyPickerCellViewModel.swift

```diff
@@ -0,0 +1,46 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
+import Result
+
+protocol SettingsCurrencyPickerCellViewModelInputs {
+  func configure(with cellValue: SettingsCellValue)
+  func didSelectCurrency(currency: Currency)
+}
+
+protocol SettingsCurrencyPickerCellViewModelOutputs {
+  var notifyCurrencyPickerCellRemoved: Signal<Void, NoError> { get }
+  var showCurrencyChangeAlert: Signal<Currency, NoError> { get }
+}
+
+protocol SettingsCurrencyPickerCellViewModelType {
+  var inputs: SettingsCurrencyPickerCellViewModelInputs { get }
+  var outputs: SettingsCurrencyPickerCellViewModelOutputs { get }
+}
+
+final class SettingsCurrencyPickerCellViewModel: SettingsCurrencyPickerCellViewModelOutputs,
+SettingsCurrencyPickerCellViewModelInputs, SettingsCurrencyPickerCellViewModelType {
+
+  public init() {
+    self.notifyCurrencyPickerCellRemoved = self.selectedCurrencyProperty.signal.ignoreValues()
+
+    self.showCurrencyChangeAlert = self.selectedCurrencyProperty.signal.skipNil()
+  }
+
+  fileprivate let selectedCurrencyProperty = MutableProperty<Currency?>(nil)
+  public func didSelectCurrency(currency: Currency) {
+    self.selectedCurrencyProperty.value = currency
+  }
+
+  fileprivate let cellTypeProperty = MutableProperty<SettingsAccountCellType?>(nil)
```

Comment:
It also seems like this input is no longer needed since we don't read from `cellTypeProperty` anywhere anymore.